tests/Doctrine/Tests/Common/Cache/RedisCacheTest.php

```diff
@@ -35,18 +35,6 @@ public function testGetRedisReturnsInstanceOfRedis()
         $this->assertInstanceOf('Redis', $this->_getCacheDriver()->getRedis());
     }
 
-    public function testSerializerOptionWithOutIgbinaryExtension()
```

Comment:
Ok, I can do that :) Any ideas? I'm very bad at that. RedisWithPresetSerializerCache ?